examples/kubernetes-mixin-example/porter.yaml

```diff
@@ -0,0 +1,39 @@
+# This is the configuration for Porter
+# You must define steps for each action, but the rest is optional
+# Uncomment out the sections below to take full advantage of what Porter can do!
```

Comment:
nit: can you remove this comment section here? It doesn't really apply to your example